src/Panel/Document.php

```diff
@@ -93,6 +93,11 @@ public static function assets(): array
 					'src'   => static::customAsset('panel.js'),
 					'type'  => 'module'
 				],
+				'plugins-mjs'      => [
```

Comment:
Can we include this asset conditionally if the file is not going to be empty? This would avoid the extra request in production.

Reply:
Yep, think so. We could probably just inline it, then it'd be one less request in the chain during development (no separate `/plugins.mjs`) and also no unnecessary code in production as we'd just "inline" an empty string there.